db/migrate/20210323171044_add_registration_metadata_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddRegistrationMetadataToUser < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
Hi, should we define this migration in `up` method in case of rollback ? 
```suggestion
  def up
```

Reply:
No in this case we add a column. We don't do data transformation.

In case of a rollback, Rails MUST be able to do the down method, and in this case, Rails is smart enough to do it.